config/crd/bases/ramendr.openshift.io_drclusterconfigs.yaml

```diff
@@ -62,6 +65,14 @@ spec:
             type: object
```

Comment:
@raaizik This is where the discrepancy is. I am wondering why the `make generate` output is different for you and the CI.

Most likely a version difference in controller-gen.